docs/en/observability/apm/known-issues.asciidoc

```diff
@@ -19,6 +19,82 @@ If applicable, exact error messages linked to this issue so users searching for
 Link to fix
 ////
 
+*Upgrading to v8.15.0 may cause APM indices to lose their lifecycle policy* +
+_Elastic Stack versions: 8.15.0_ +
+_Fixed in Elastic Stack version 8.15.1_
+
+// The conditions in which this issue occurs
+The issue only occurs when _upgrading_ the {stack} to 8.15.0.
+The issue does _not_ occur when creating a _new_ cluster using 8.15.0.
+The issue also does not occur if a custom ILM policy is configured using a custom component template.
+
+// Describe why it happens
+In 8.15.0, APM Server switched to use Data stream Lifecycle to manage data retention
+for APM indices for new deployments as well as for upgraded deployments with default lifecycle
+configurations. Unfortunately, since any data stream created before 8.15.0 does not have data
+stream lifecycle configuration, such existing data streams become unmanaged for default
```

Comment:
```suggestion
configurations. Unfortunately, since any data stream created before 8.15.0 does not have a data
stream lifecycle configuration, such existing data streams become unmanaged for default
```